SupportClient/SupportClient/Views/Cells/BugsRequestsTableViewCell.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BugsRequestsTableViewCell.swift
+//  SupportClient
+//
+//  Created by Yana Sang on 5/8/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+class BugsRequestsTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = "bugsRequestsTableViewCell"
+
+    private let tagsLabel = UILabel()
+    private let timeLabel = UILabel()
+    private let messageLabel = UILabel()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    func setupViews() {
+        tagsLabel.font = UIFont.systemFont(ofSize: 12, weight: .regular)
```

Comment:
Is SFPro the system font?

Reply:
it's the san francisco fonts, but i think which variant it uses depends on the size
for a small font size i'm pretty sure it is SF pro text